docs/faq-user.md

```diff
@@ -20,37 +20,32 @@ While there is a delay before you can access your previously delegated tokens, t
+The base annual reward rate is 8% of the total supply, resulting in a current APY of approximately 15% on the Mainnet (because only a portion of the total supply is currently staked and participating in the reward distribution). The APY might decrease over time as more tokens get staked on the network.
 
 ## I see some validators with big stake producing blocks, and some smaller ones not. Does that mean only the bigger ones are getting rewards?
 
-No. On Casper Mainnet, everyone gets seigniorage according to their stake, regardless of whether you've produced blocks or not. The only thing a block proposer gets, is to keep the transaction fees associated with transfers and deploys in the block they proposed. As you may know, transfers cost only 10000 motes (0.00001 CSPR), so these fees are not substantial on a transfer-by-transfer basis.
+No. On Casper Mainnet, everyone gets seigniorage according to their stake, regardless of whether you've produced blocks or not. The only thing a block proposer gets, is to keep the transaction fees associated with transfers and deploys in the block they proposed. As you may know, transfers cost only 100000000 motes (0.1 CSPR), so these fees are not substantial on a transfer-by-transfer basis.
 
 The block proposer is chosen also based on odds relative to stake, so if you have 0.1% of stake on the network, the odds that you'll get chosen as the block proposer are very small. This has nothing to do with seigniorage.
 
+As per the new design of validator rewards on Casper 2.0, some volatility in rewards is expected in the short run. However, all validator rewards should average to an amount proportional to their total stake in the long run.
+
 ## Some top validators have a commission rate of 100%. What does that mean?
 
 Validators with Delegation rate of 100 are ones that either don't want to or are restricted from acting as a staking service. You will earn nothing from them.
 
-## Which algorithm should I choose while creating a new account on Signer?
-
-There are two different encryption algorithms that the Casper Network supports.
-
-* `Ed25519` is fast and provides a high level of security with foolproof session keys, among other features. It is the default algorithm used while creating new accounts by the Casper client.
-* `Secp256k1` is an efficient encryption algorithm, also used by Bitcoin and Ethereum. If you would like to derive Ethereum or Bitcoin keys in the future using your private key, then go with the Secp256k1 algorithm.
-
 ## How can I delegate my tokens?
 
-An easy, web-based staking UI is now available on [CSPR.Live](https://cspr.live). You may follow these steps to delegate your tokens:
+An easy, web-based staking UI is available on [CSPR.Live](https://cspr.live). You may follow these steps to delegate your tokens:
```

Comment:
Let's link directly to https://cspr.live/delegate-stake? Wdyt?

Reply:
@mssteuer Makes sense. Done. :heavy_check_mark: